app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2><%= t "devise.confirmations.resend_instructions" %></h2>
+<%= form_for(resource, as: resource_name, url: user_confirmation_path(resource_name), html: { method: :post }) do |f| %>
```

Comment:
resource ở đây là gì, và tại sao lại nên define là resource mà ko dùng trự tiếp model

Reply:
Dạ resource đại diện cho model được truyền vào cho `devise for` trong routes.rb
vd: `devise_for :users` thì resource sẽ là instance của class User

Reply:
:v ko nhé 

để lỡ ko phải dự án của em chri dùng 1 model user mà dùng 3 -4 cái model để phân quyền thì sao :D

Reply:
Dạ